CONTRIBUTING.md

```diff
@@ -135,6 +137,9 @@ before it will be merged. The basic squashing workflow is shown below.
     # Save and quit.
     git push -f # (force push to GitHub)
 
+The commit message should be updated so that it doesn't include all the
+messages from the interim commits.
```

Comment:
IMO, this requirement is not suitable for all cases.

Reply:
Interesting! Would you (or someone else) care to elaborate? 
Personally I don't really know, but it seems like something that would benefit from a set guideline.

Reply:
e.g., 869193f5a6398bb9e4a6f4a7aeb8d2203193c80f
That commit was squashed from two ones. And its message _does_ include all the messages from the squashed commits.

Reply:
I think I see what you are saying; there can be cases where just listing the interim commits happens to form a coherent message. 

New suggestion:
`Please update the resulting commit message if needed, it should read as a coherent message. In most cases this means that you should not just list the interim commits.`

Reply:
LGTM